relay-event-normalization/src/normalize/span/tag_extraction.rs

```diff
@@ -365,6 +365,12 @@ pub fn extract_tags(
             } else {
                 None
             }
+        } else if span.origin.as_str() == Some("auto.db.supabase") {
+            scrubbed_description.as_deref().map(|s| {
+                s.trim_start_matches("from(")
```

Comment:
Should we enforce that this description always starts with `from(`?

Reply:
Thanks, this actually surfaced a bug, see follow-up here: https://github.com/getsentry/relay/pull/3156